lib/client-assets.php

```diff
@@ -189,18 +176,18 @@ function wc_admin_print_script_settings() {
 	 * `wcAssetUrl` can be used in its place throughout the codebase.
 	 */
 	$settings = array(
-		'adminUrl'              => admin_url(),
-		'wcAssetUrl'            => plugins_url( 'assets/', WC_PLUGIN_FILE ),
-		'wcAdminAssetUrl'       => plugins_url( 'images/', wc_admin_dir_path( 'wc-admin.php' ) ), // Temporary for plugin. See above.
-		'embedBreadcrumbs'      => wc_admin_get_embed_breadcrumbs(),
-		'siteLocale'            => esc_attr( get_bloginfo( 'language' ) ),
-		'currency'              => wc_admin_currency_settings(),
-		'orderStatuses'         => format_order_statuses( wc_get_order_statuses() ),
-		'stockStatuses'         => wc_get_product_stock_status_options(),
-		'siteTitle'             => get_bloginfo( 'name' ),
-		'trackingEnabled'       => $tracking_enabled,
-		'dataEndpoints'         => array(),
-		'l10n'                  => array(
+		'adminUrl'         => admin_url(),
+		'wcAssetUrl'       => plugins_url( 'assets/', WC_PLUGIN_FILE ),
+		'wcAdminAssetUrl'  => plugins_url( 'images/', wc_admin_dir_path( 'wc-admin.php' ) ), // Temporary for plugin. See above.
+		'embedBreadcrumbs' => wc_admin_get_embed_breadcrumbs(),
+		'siteLocale'       => esc_attr( get_bloginfo( 'language' ) ),
+		'currency'         => wc_admin_currency_settings(),
+		'orderStatuses'    => format_order_statuses( wc_get_order_statuses() ),
+		'stockStatuses'    => wc_get_product_stock_status_options(),
+		'siteTitle'        => get_bloginfo( 'name' ),
+		'trackingEnabled'  => 'yes' === get_option( 'woocommerce_allow_tracking', 'no' ),
```

Comment:
I think we should remove this in favour of letting Core handle `trackingEnabled` logic. One reason is the `woocommerce_apply_user_tracking` filter allows plugins to turn off tracking despite what the user's `woocommerce_allow_tracking` option value is.

https://github.com/woocommerce/woocommerce/blob/37d3fff3d6c9eeda66e32f1c29bcf3e6fe543a46/includes/tracks/class-wc-site-tracking.php#L25-L27

Reply:
@psealock I went back and forth on this, but in light of the `woocommerce_apply_user_tracking` filter, you're definitely right.  Removed- thanks for noting this!